code/modules/economy/cael/related_accounts.dm

```diff
@@ -150,7 +151,7 @@
 	if(!parent_account || !money) // Nothing to escrow!
 		return
 
-	var/solvency = max(1, parent_account.money / parent_account.child_totals)
+	var/solvency = min(1, parent_account.money / parent_account.child_totals)
```

Comment:
I didn't spot that earlier, but is it possible for ``parent_account.child_totals`` to be 0? Because that would cause a division by 0.

Reply:
If this account has no money, the proc returns above. So it should not occur unless something has gone wrong, in which case the runtime shouldn't be suppressed.

Reply:
fair enough